src/app/Instance/CreateInstance/CreateInstance.tsx

```diff
@@ -65,26 +79,46 @@ const CreateInstance = (props: CreateInstanceProps): JSX.Element => {
     [existingInstanceName, validate]
   );
 
-  useEffect(() => {
-    if (existingInstanceName) {
-      validate();
-    }
-  }, [existingInstanceName, validate]);
-
   useEffect(() => {
     if (isModalOpen) {
       setName("");
       setError(null);
+      setGenericError(null);
     }
   }, [isModalOpen]);
 
+  useEffect(() => {
+    if (createBridgeError && axios.isAxiosError(createBridgeError)) {
+      if (createBridgeError.response?.status === 500) {
+        setGenericError(createBridgeError.response?.statusText);
+      } else if (
+        isServiceApiError(createBridgeError) &&
+        getErrorCode(createBridgeError) === APIErrorCodes.ERROR_1
+      ) {
+        setExistingInstanceName(newBridgeName);
+      }
+    }
```

Comment:
Should we check first for managed errors (`APIErrorCodes.ERROR_1` only in this case) and treat everything else like a generic error? So not checking directly for a `500`. This could help us having a basic error handling for any scenario we didn't think about.

What you think?